.gitlab-ci.yml

```diff
@@ -421,6 +421,27 @@ test:pkgs:workstation-tools:
     - test -x /usr/bin/mender-cli
     - mender-cli --version
 
+test:pkgs:workstation-tools:trixie:
```

Comment:
This job is exactly like above except for:
* The job name
* `needs`
* `image`

Please consider having a hidden job for example `.test:pkgs:workstation-tools` and then these two jobs extending it. See how it is done elsewhere.

Good initiative adding the tests, though!

Reply:
I have added the tests with the common part, hope it is according to CI gods rules :D